roles/kube-node-common/files/cloud-init-report-alert.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Notify function
+# Sends notification to alertmanager
+# $1: The alertmanager alerts url
+# $2: Severity: warning,critical,error
+# $3: Summary: Description about the error
+notify()
+{
+    ALERT_NAME="cloud-init"
+    ALERT_MANAGER_HOSTNAME="$1"
+    URL="http://${ALERT_MANAGER_HOSTNAME}/api/v2/alerts"
```

Comment:
Since the `ALERT_MANAGER_HOSTNAME` can be empty, shouldn't we check it's value before trying to use it?

Reply:
You're right. My thinking while developing was...

Alert manager will be requested ONLY on errors. So, if there is no `ALERT_MANAGER_HOSTNAME` defined, it does not matter to have an error requesting `http:///api/v2/alerts`.

Reply:
If there are no errors with the empty string, we can leave it as is.

Reply:
I mean... If something goes wrong, You will be notify only when the `ALERT_MANAGER_HOSTNAME` is defined and **correct**.

If you send a typographical error to the `ALERT_MANAGER_HOSTNAME` variable or leave it empty, you will not be notified in any way.

Also, all of this bash logic has to be moved to `fury-agent`. We have time to refactor it.

Reply:
Agreed, that's fine, checking and crashing or trying to send and crashing have the same outcome.